### ISSUE TITLE
Recent Pull created issue with chemprop_predict

### ISSUE THREAD

**Issue by daverdude** [chemprop/chemprop#148]
I recently encountered the issue with find_sim_mols.py, which was remedied by pull request #144.

However, after completing that recent pull I have now encountered a new issue with predict.py using previously trained models. I receive the following errors. I have gone ahead and removed the --no_features_scaling flag, tried to parallelize the features generations, but then I get met with the following error. AttributeError: 'PredictArgs' object has no attribute 'overwrite_default_atom_features'

(chemprop) [daviddelgadillo@localhost chemprop]$ chemprop_predict --test_path data/ecoli.csv --features_path features/ecoli.npz --no_features_scaling --checkpoint_dir 0405_broad_inhibition_20folds --preds_path pred/ecoli_antibiotic_20folds.csv
WARNING:root:No normalization for BCUT2D_MWHI
WARNING:root:No normalization for BCUT2D_MWLOW
WARNING:root:No normalization for BCUT2D_CHGHI
WARNING:root:No normalization for BCUT2D_CHGLO
WARNING:root:No normalization for BCUT2D_LOGPHI
WARNING:root:No normalization for BCUT2D_LOGPLOW
WARNING:root:No normalization for BCUT2D_MRHI
WARNING:root:No normalization for BCUT2D_MRLOW
Loading training args
Traceback (most recent call last):
  File "/home/daviddelgadillo/anaconda3/envs/chemprop/bin/chemprop_predict", line 33, in <module>
    sys.exit(load_entry_point('chemprop', 'console_scripts', 'chemprop_predict')())
  File "/home/daviddelgadillo/chemprop/chemprop/train/make_predictions.py", line 206, in chemprop_predict
    make_predictions(args=PredictArgs().parse_args())
  File "/home/daviddelgadillo/chemprop/chemprop/utils.py", line 437, in wrap
    result = func(*args, **kwargs)
  File "/home/daviddelgadillo/chemprop/chemprop/train/make_predictions.py", line 47, in make_predictions
    raise ValueError('If scaling of the additional features was done during training, the'
ValueError: If scaling of the additional features was done during training, thesame must be done during prediction.

**Comment by cjmcgill** [comment]
Some new arguments were introduced in #137 that are part of training and then referenced during prediction. Models created before #137 and predictions done after #137 aren't compatible anymore.

I am working on a patch that will make them backwards compatible.

**Comment by cjmcgill** [comment]
It is not necessary to for testing but it would be helpful: when did you generate the model files that you want to make predictions from? Which is to say, what version of chemprop did you use/when did you last update it before making those models?

**Comment by daverdude** [comment]
Ahhh okay. That model, in particular, was the model used for the Antibiotic discovery paper published in Cell (created by Kyle, I'm assuming). So I am sure that it was created well before the #137 update. I will run some models that I created last month to see if those work.
